src/app/docs/summary-action-bar/summary-action-bar-modal-docs.component.scss

```diff
@@ -0,0 +1,3 @@
+sky-key-info {
```

Comment:
Since this is demo code, is there a comment on why this is necessary or when to use this?

Reply:
Yep, I went ahead and commented all the places where this appears.